ui/src/components/entity/citation/PaperSummary.tsx

```diff
@@ -199,7 +185,14 @@ export default class PaperSummary extends React.PureComponent<Props, State> {
         </div>
 
         <div className="paper-summary__library-error">
-          {this.state.errorMessage && this.state.errorMessage}
+          {!!this.state.errorMessage && this.state.errorMessage}
+          {this.state.showLoginMessage && (
+            <React.Fragment>
+              Before you can save papers to your library, you must be logged into Semantic Scholar.
+              Visit <ExternalLink href="https://www.semanticscholar.org" allowReferrer>https://semanticscholar.org</ExternalLink> to log in.
```

Comment:
propose a slight tweak, link to https://www.semanticscholar.org/me/library instead so people will see a sign in page right away

Reply:
Sounds good. Since that makes the URL longer I'll also just rename the link to "Semantic Scholar" in the UI.